components/components/navigation/menu--main/menu-main.css

```diff
@@ -2,3 +2,14 @@
  * @file
```

Comment:
Why do you use both menu--main and menu-main on the file name? (menu--main/menu-main.css)

Reply:
That's fixed now @ckrina